dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -678,7 +678,7 @@ void CommandBarFlyoutCommandBar::UpdateTemplateSettings()
             collapsedWidth = static_cast<float>(expandedWidth);
         }
 
-        flyoutTemplateSettings->WidthExpansionDelta(collapsedWidth - expandedWidth);
+        flyoutTemplateSettings->WidthExpansionDelta(0);
```

Comment:
I don't understand this change, but if @llongley or someone else does that is good enough.

Reply:
I thought there was a need for a nonzero value for this, but Karen said that she tested all of the scenarios and they all continue to look right, so if that's the case it seems OK to me.

Reply:
When PrimaryCommands are being loaded dynamically, it causes a weird crop issue that is caused by a negative `WidthExpansionDelta`. Setting it to 0 fixes this specific use case. Upon investigation, it seems a non-zero `WidthExpansionDelta` is only for when SecondaryCommands is collapsed, and the width of primaryCommands is less than the width of SecondaryCommands, so the flyout expands to the full width when opening the secondaryCommands. 

However, setting it to 0 doesn't seem to break that scenario. This may be because CBF no longer has the expanding animation. I manually all scenarios with and without the `WidthExpansionDelta` change, and it doesn't change or break anything.

Reply:
The CBF does have the expanding animation still though, there is a button for it on the CommandBarFlyout test page labeled "Show expanding CommandBarFlyout"

This is the behavior before this change:


https://user-images.githubusercontent.com/7758786/141835292-2f6c4da9-cf1f-42d2-9a5f-f2c4cec69916.mp4


and this is the behavior after the change

https://user-images.githubusercontent.com/7758786/141831780-7a9e8b14-bb52-47c8-bdaf-7fea2d885109.mp4

Reply:
Ah I now see a subtle expanding animation that is missing from the update. I think the appropriate action is to revert the hardcoding of ExpansionWidthDelta, and fix the dynamically adding primary buttons in a separate fix. It is beyond the scope of the original accessibility bug to begin with.